cobc/ChangeLog

```diff
@@ -1,4 +1,12 @@
 
+2022-12-16  Nicolas Berthier <nicolas.berthier@ocamlpro.com>
```

Comment:
enum literal type refactoring is missing here

Reply:
done